doc/src/manual/control-flow.md

```diff
@@ -567,17 +574,16 @@ below all interrupt the normal flow of control.
 For example, the [`sqrt()`](@ref) function throws a [`DomainError`](@ref) if applied to a negative
 real value:
 
-```julia
+```jldoctest
 julia> sqrt(-1)
 ERROR: DomainError:
 sqrt will only return a complex result if called with a complex argument. Try sqrt(complex(x)).
- in sqrt(::Int64) at ./math.jl:278
```

Comment:
It can be noted that the `[...]` is actually rendered too so it is implicit that there are things hidden. The above case renders to this:
![backtrace](https://cloud.githubusercontent.com/assets/11698744/22241041/fd6a6658-e21d-11e6-993a-ee1b4f0a9898.png)

Note also that it is not consistent whether the backtrace is shown or not across the manual. What do other people have to say?